src/config/env/index.js

```diff
@@ -9,11 +9,13 @@ const config = require(`./${env}`);
 const defaults = {
     userRoles: ['guest', 'user:read', 'user:write', 'user', 'admin'],
     root: path.join(__dirname, '/..'),
+    cleanUploads: false,
+    uploadsFolder: './uploads',
```

Comment:
I agree, but here I have to put "path.join(__dirname, '../../../uploads')" to get the same result. It's too many "../", isn't it?